src/reducers/transactionReducer.js

```diff
@@ -154,15 +156,27 @@ function deleteOutput(state, action) {
 
 function updateFeeRate(state, action) {
   const feeRateString = action.value;
-  const feeRateError = validateFeeRate(feeRateString);
+
+  // Gets the error type. Useful for conditionally displaying errors.
+  const feeRateError = checkFeeRateError(feeRateString);
+
+  // Get the fee. Ignore rate-too-high errors because this value creates
+  // problems when trying to author a CPFP.
   const fee =
-    feeRateError === ""
+    feeRateError === null &&
```

Comment:
Doesn't this need to be an or statement to ignore rate-too-high errors? If I'm reading this correctly fees will only be set if feeRateError is null
```
feeRateError === null ||
feeRateError === FeeValidationError.FEE_RATE_TOO_HIGH
```

Reply:
Thats a good point. We could set the feeRateErrorMessage when fee is an empty string to avoid the disconnect, but I'm also fine with what you are doing now.